infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/CommonPVCStrategy.java

```diff
@@ -105,12 +112,14 @@ public CommonPVCStrategy(
       @Named("che.infra.kubernetes.pvc.precreate_subpaths") boolean preCreateDirs,
       @Named("che.infra.kubernetes.pvc.storage_class_name") String pvcStorageClassName,
       @Named("che.infra.kubernetes.pvc.wait_bound") boolean waitBound,
+      @Nullable @Named("che.infra.kubernetes.namespace.default") String defaultNamespaceName,
```

Comment:
why is this needed? Looks like it's not used anywhere.

Reply:
Oops, it was from a test that was removed. Fixed in the latest commit.